scripts/run_altvr_benchmark.sh

```diff
@@ -0,0 +1 @@
+ cargo run --release --features=runtime-benchmarks -- benchmark --chain dev  --execution=wasm --extrinsic="*" --pallet=ternoa_altvr --steps=50 --repeat=20 --heap-pages=4096 --output .
```

Comment:
Kudos to you for remembering to add this file.

Reply:
👍